assets/js/modules/analytics-4/datastore/__fixtures__/report.json

```diff
@@ -0,0 +1,1961 @@
+{
```

Comment:
This report fixture is not being used anywhere, I would have expected to see it used in a test to compare the mock output...

It could also be worth adding and testing against a fixture which only uses a single date range, and also including a wider variety of data types...

Reply:
@techanvil I updated the tests but maybe I'll need some help after you review about possible tests to write. Right now I'am mostly checking the format of `dimensionHeaders`, `metricsHeaders`, etc